src/main/java/creativei/manager/impl/ReminderManagerImpl.java

```diff
@@ -38,4 +40,18 @@ public ResponseObject create(ReminderVo reminderVo) {
             return ResponseObject.getResponse(ExceptionType.GENERAL_ERROR.getMessage(),ExceptionType.GENERAL_ERROR.getCode());
         }
     }
+
+    @Override
+    public ResponseObject getReminderByDateRange(String to, String from) {
+        logger.info("Get Reminder by date range Method");
+        List<Reminder> reminders= null;
+        try {
```

Comment:
Convert string to dates in manager layer. 
1. Sting can be null not "null" as wrapped in json.
2. If fromDate is null, get all inquries before toDate(including)
3. If toDate is null, get all greater than equal to fromDate.
4. If fromDate is greater than toDate, send error response "invalid request param."(reuse same error we have)
5. If string date is not able to convert to date object, send invalid request param error.

Reply:
string conversion added in manager layer in commit cd46f64